blackbox/docs/sql/scalar.txt

```diff
@@ -1639,6 +1641,76 @@ Example::
     +----------------+
     SELECT 1 row in set (... sec)
 
+``CURRENT_USER``
+----------------
+
+The ``CURRENT_USER`` system information function returns the name of the 
+current connected user.
+
+Returns: ``string``
+
+Synopsis::
+
+    CURRENT_USER 
+
+Example::
+
+    cr> select current_user;
+    +--------------+
+    | current_user |
+    +--------------+
+    | crate        |
+    +--------------+
+    SELECT 1 row in set (... sec)
+
+``USER``
+--------
+
+Equivalent to ``CURRENT_USER``.
+
+Returns: ``string``
+
+Synopsis::
+
+    USER 
+
+Example::
+
+    cr> select user;
+    +--------------+
+    | current_user |
+    +--------------+
+    | crate        |
+    +--------------+
+    SELECT 1 row in set (... sec)
+
+``SESSION_USER``
+----------------
+The ``SESSION_USER`` system information function returns the name of the
```

Comment:
Because `SESSION_USER` is not _the same_ as `CURRENT_USER`. See the difference in result column name.